### ISSUE TITLE
Server Error for Controllers

### ISSUE THREAD

**Issue by JoshuaMicallefYBSU** [vatsimnetwork/nattrak#166]
Controllers getting 500 errors for
- Transmit Clearance
- Login Screen

Not sure what's wrong but the site is functionally broken with most Controllers unable to use it.

**Comment by lieselwd** [comment]
Working on it

**Comment by williammck** [comment]
Waiting on an upstream outage then will be able to deploy the fixes.

**Comment by williammck** [comment]
@JoshuaMicallefYBSU Fixed version of the site should be deployed. We'll monitor for further errors.

Always feel free to get in touch with us if you notice an issue. This went unnoticed for two weeks, and multiple controllers ran into it without any sort of message to us.

**Comment by JoshuaMicallefYBSU** [comment]
Thanks mate, appreciate the quick fix on this.

What is the best way to contact you guys? I'm in the dev discord but can not see any channels which would be for notifying such issues.

**Comment by lieselwd** [comment]
Just a ping in the general chat works :)

**Comment by JoshuaMicallefYBSU** [comment]
Server Error occured when transmitting a clearance just now :)

**Comment by williammck** [comment]
Luckily it sent me an email regarding the error this time... Taking a look.

**Comment by williammck** [comment]
@JoshuaMicallefYBSU Give it a try now.

**Comment by JoshuaMicallefYBSU** [comment]
Thanks guys, seems to have resolved the issues!

A question for @lieselwd here, as another issue seems to have appeared. A lot of our controllers do not appear in the https://nattrak.vatsim.net/administration/controllers list and as such they get the following error.

<img src="https://cdn.discordapp.com/attachments/836707337829089322/1312695167995871373/image.png?ex=674d6e24&is=674c1ca4&hm=42c3c730b0f0c456f63b6fb611b41c5a088e03bee2d51847446b36c0525b8770&">

Any idea on a simple solution to this that doesnt require manually adding the user? This seems to be a relativley new issue.
